pkgs/development/dotnet-modules/python-language-server/create_deps.sh

```diff
@@ -0,0 +1,29 @@
+
+# To use this script, clone the repo and set this variable to the checkout path:
+CHECKOUT_PATH=/home/tom/tools/microsoft-python-language-server/
+
+######################################################
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
```suggestion
SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
```

Reply:
Hmm, why this change? I've been using this snippet for a long time and I don't think it can print anything.

Reply:
It's what other similar scripts seem to do, such as `pkgs/applications/science/electronics/kicad/update.sh` and `pkgs/development/tools/purescript/spago/update.sh`. Really it's a "better safe than sorry" kind of situation, not something imperative that needs to be dealt with. Feel free to disregard!

Reply:
Oh you're right, your version is the one given [here](https://stackoverflow.com/questions/59895/how-to-get-the-source-directory-of-a-bash-script-from-within-the-script-itself). Will change. Wonder how mine diverged...